CHANGELOG.md

```diff
@@ -57,6 +57,14 @@ If you’re calling `iff` from your own Sass, you should use [Sass's native `if`
 
 This change was introduced in [pull request #2409: Remove deprecated `iff` Sass function](https://github.com/alphagov/govuk-frontend/pull/2409).
 
+#### Remove deprecated `.govuk-tag--inactive` class
+
+We've removed the `.govuk-tag--inactive` class which we deprecated in [GOV.UK Frontend v3.6.0](https://github.com/alphagov/govuk-frontend/releases/tag/v3.6.0).
+
+Replace any use of this class with the `.govuk-tag--grey` class.
+
+This change was introduced in [pull request #2417: Remove deprecated `.govuk-tag--inactive class`](https://github.com/alphagov/govuk-frontend/pull/2417).
```

Comment:
Really minor, but suggest we avoid prefixing the class names with a period in the release notes because:

- technically the dot is part of the CSS selector rather than the class name
- wherever users are 'using' the class (in the classes in the HTML, or when passing classes to the Nunjucks macro) they wouldn't be including the `.`, so including it in the instructions may be confusing.

Appreciate this is inconsistent with, for example, the release notes for 3.6.0, but still think it's worth changing.

```suggestion
#### Remove deprecated `govuk-tag--inactive` class

We've removed the `govuk-tag--inactive` class which we deprecated in [GOV.UK Frontend v3.6.0](https://github.com/alphagov/govuk-frontend/releases/tag/v3.6.0).

Replace any use of this class with the `govuk-tag--grey` class.

This change was introduced in [pull request #2417: Remove deprecated `govuk-tag--inactive class`](https://github.com/alphagov/govuk-frontend/pull/2417).
```